pkgs/os-specific/linux/kbd/keymaps.nix

```diff
@@ -24,7 +24,8 @@
 
     src = fetchurl {
       name = "neo.map";
-      url = "https://svn.neo-layout.org/linux/console/neo.map?r=${version}";
+      url = "https://raw.githubusercontent.com/neo-layout/neo-layout/"
+          + "a0dee06fed824abfad658b7f10e6d907b270be0a/linux/console/neo.map";
```

Comment:
shouldn't that depend on version too ? or just give the github link

Reply:
It depends on specific github commit which corresponds to the version 2476 migrated from svn. This url points to the raw version of github file https://github.com/neo-layout/neo-layout/blob/a0dee06fed824abfad658b7f10e6d907b270be0a/linux/console/neo.map .